queue/queue.py

```diff
@@ -3,13 +3,16 @@ def __init__(self):
     self.size = 0
     # what data structure should we
     # use to store queue elements?
-    self.storage = 
+    self.storage = []
 
   def enqueue(self, item):
-    pass
+    return self.storage.insert(0, item)
   
   def dequeue(self):
-    pass
+    if len(self.storage):
+      return self.storage.pop()
 
```

Comment:
Everything looks great for Data Structures so far Abi! 😁 If you are curious to dig deeper, the hackerrank videos are very helpful!